tests/unit/admin/test-class-wc-rest-payments-settings-controller.php

```diff
@@ -255,6 +266,30 @@ public function test_get_settings_fails_if_user_cannot_manage_woocommerce() {
 		remove_filter( 'user_has_cap', $cb );
 	}
 
+	public function test_get_settings_without_error_when_faulty_enabled_payment_methods() {
+		$this->upe_gateway->update_option(
+			'upe_available_payment_method_ids',
```

Comment:
Is this option the correct one?

Reply:
Thanks for pointing this out. It may have happened while I referred to an earlier test. I understand now that we are wanting to test the `gateway` and not just `upe_gateway` response. I have changed this to use `gateway`. I hope this is fine now. 

[b547683](https://github.com/Automattic/woocommerce-payments/pull/5388/commits/b547683f01d8588a4ba746b89ac1b3b90ec92fb6)

Reply:
@kalessil Just to confirm, may I mark this as resolved? This is among my first tests and hence want to be safe :)). Thanks.